DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: secuTrialR
 Type: Package
 Title: Handling of data from the clinical data management system secuTrial 
-Version: 0.6.3
+Version: 0.6.6
```

Comment:
what happened to 0.6.5?

Reply:
@markomi  is working on something which will be 0.6.5